js/aktivitetskrav/data/aktivitetskrav/aktivitetskrav.js

```diff
@@ -1,4 +1,4 @@
-import { BEKREFTER_AKTIVITETSKRAV, AKTIVITETSKRAV_BEKREFTET, BEKREFT_AKTIVITETSKRAV_FEILET } from '../actions/actiontyper';
+import { BEKREFTER_AKTIVITETSKRAV, AKTIVITETSKRAV_BEKREFTET, BEKREFT_AKTIVITETSKRAV_FEILET } from '../../../actions/actiontyper';
```

Comment:
Samme som den andre PRen. Dett hadde vært fint hvis actionsene lå sammen med action creatorene som bruker dem. Da blir det letter å se hva som henger sammen, og man slipper å ha lange pather for å importere.